compose/config/environment.py

```diff
@@ -10,27 +10,32 @@
 
 from ..const import IS_WINDOWS_PLATFORM
 from .errors import ConfigurationError
+from .errors import EnvFileNotFound
 
 log = logging.getLogger(__name__)
 
 
 def split_env(env):
     if isinstance(env, six.binary_type):
         env = env.decode('utf-8', 'replace')
+    key, value = None, None
     if '=' in env:
-        return env.split('=', 1)
+        key, value = env.split('=', 1)
     else:
-        return env, None
+        key = env
+    if ' ' in key:
```

Comment:
`if string.whitespace in  key:` would be better and allow us to also catch tabs and the like.

Reply:
`string.whitespace` is string, so I did something like:
```
for k in key:
  if k in string.whitespace:
```